SmartDeviceLink/SDLNotificationConstants.h

```diff
@@ -82,6 +82,7 @@ extern SDLNotificationName const SDLTransportDidConnect;
 extern SDLNotificationName const SDLDidReceiveError;
 extern SDLNotificationName const SDLDidReceiveLockScreenIcon;
 extern SDLNotificationName const SDLDidBecomeReady;
+extern SDLNotificationName const SDLDidProjectionViewUpdate;
```

Comment:
I think I would still prefer this to be `SDLDidUpdateProjectionView` as noted in the last review. The style here tends to be `SDL[Did/Will/etc.][verb][object]`